utils/core-modules/test/contracts/modules/FeatureFlagModule.test.ts

```diff
@@ -140,4 +140,27 @@ describe('FeatureFlagModule', function () {
       assertBn.equal(await SampleFeatureFlagModule.getFeatureFlaggedValue(), 25);
     });
   });
+
+  describe('set denyAll for a feature flag', async function () {
```

Comment:
add test to make sure only owner can set this

Reply:
added